server/api/db/sqldb/models.py

```diff
@@ -769,6 +774,20 @@ class TimeWindowTracker(Base, mlrun.utils.db.BaseModel):
         def get_identifier_string(self) -> str:
             return f"{self.key}"
 
+    class FunctionSpecFoo(Base, mlrun.utils.db.BaseModel):
+        __tablename__ = "function_spec_foo"
```

Comment:
```suggestion
        __tablename__ = "function_spec_foos"
```
We usually use plurals for table names